src/extension.ts

```diff
@@ -98,6 +79,21 @@ export function activate(context: ExtensionContext) {
         languageClient.onNotification('sourcery/vscode/viewProblems', () => {
             commands.executeCommand('workbench.actions.view.problems');
         });
+
+        languageClient.onNotification('sourcery/vscode/afterInstall', () => {
+            const readmePath = Uri.file(
+                path.join(context.extensionPath, 'welcome-to-sourcery.py')
+            );
+            window.showTextDocument(readmePath);
+            const result = window.showInputBox({
+                placeHolder: 'Sourcery Token',
+                prompt: 'Get your token from https://sourcery.ai/signup',
+                ignoreFocusOut: true
+            });
+            result.then(function (value) {
+                workspace.getConfiguration('sourcery').update('token', value, true)
+            });
```

Comment:
As far as I remember, the outcome of the discussion was:
We still show the input box for the token, because it makes it more convenient for already registered users to enter a token.